examples/starter/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="jotai examples" />
+    <title>Jotai Examples | Starter</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="module" src="/src/app.tsx"></script>
```

Comment:
On second thought, let's make this as the same as others:
```suggestion
    <div id="root"></div>
    <script type="module" src="/src/index.tsx"></script>
```

and rename app.tsx to index.tsx